stage0_starting_script/face_detect_skimage.py

```diff
@@ -1,22 +1,33 @@
+
+import PIL.Image
+from PIL.ExifTags import TAGS
 from skimage import data
 from skimage.feature import Cascade
-
 import matplotlib.pyplot as plt
 from matplotlib import patches
+import imageio
+import sys
+from os.path import join
 
 # Load the trained file from the module root.
 trained_file = data.lbp_frontal_face_cascade_filename()
 
 # Initialize the detector cascade.
 detector = Cascade(trained_file)
 
-img = data.astronaut()
+if len(sys.argv) > 1:
+    image_path = sys.argv[1]
+else:
+    image_path = join("sample_images", "IMG-0311_xmas_2020.JPG")
+    # image_path = join("sample_images", "owls.jpg")
+
+img = imageio.imread(image_path)
```

Comment:
Matplotlib can do this too!
```suggestion
img = plt.imread(image_path)
```

Reply:
Done. Thanks.